src/booking/booking.service.ts

```
@@ -124,12 +161,13 @@ export class BookingService {
-    const updatedBooking = await this.bookingRepository.update(id, { status: BookingStatus.COMPLETED });
-
-    // Make driver available again
-    await this.driverService.update(driverId, { isAvailable: true });
-
-    return updatedBooking;
+    try {
+      const updatedBooking = await this.bookingRepository.update(id, { status: BookingStatus.COMPLETED });
+      await this.driverService.update(driverId, { isAvailable: true });
+      return updatedBooking;
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)